config/layout-renderers.json

```diff
@@ -1791,7 +1791,7 @@
             "NLog.Web",
             "NLog.Web.AspNetCore"
         ],
-        "description": "ASP.NET MVC controller name",
+        "description": "ASP.NET Lookup MVC 'controller' route parameter value",
```

Comment:
I like to have "name" somewhere  for the search results. Maybe in the keywords otherwise? ("Routing" is also a good keyword here I think)

Reply:
Updated the description to say Controller name

Reply:
Thx